docs/monitoring/monitoring-metricbeat.asciidoc

```diff
@@ -0,0 +1,117 @@
+[role="xpack"]
+[[monitoring-metricbeat]]
+=== Monitoring {kib} with {metricbeat}
+
+beta[] In 6.4 and later, you can use {metricbeat} to collect and ship data about 
+{kib}, rather than routing it through {es} as described in <<monitoring-kibana>>. 
+
+. Enable the collection of monitoring data in the {stack}. Set 
```

Comment:
The "Configuring Monitoring in Kibana" doc which lists the old/current way of monitoring has this line in it:

> To learn about monitoring in general, see Monitoring the Elastic Stack.

Do you think we should add this line in this doc as well. perhaps for users who come to this doc without reading the other one?